yt/visualization/fits_image.py

```diff
@@ -250,8 +259,7 @@ def __init__(self, data, fields=None, length_unit=None, width=None,
                     if value is not None:
                         hdu.header[key] = float(value.value)
                         hdu.header.comments[key] = "[%s]" % value.units
-                if self.current_time is not None:
-                    hdu.header["time"] = self.current_time
+                hdu.header["time"] = float(self.current_time.value)
```

Comment:
Is it worth having the units of current time here as header comments like you did above, or is that unnecessary?

Reply:
Good question--it should always be in the time units of the file stored in `header["TUNIT"]`, so it shouldn't be necessary to include it in the comments.